specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/serverVulnerabilityAssessments.json

```diff
@@ -0,0 +1,338 @@
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Security/ServerVulnerabilityAssessments": {
+            "get": {
+                "x-ms-examples": {
+                    "Get a list of server vulnerability assessments on a resource. Though this API returns a list, Currently Microsoft.Security only supports a single default type of server vulnerability assessment": {
+                        "$ref": "./examples/ServerVulnerabilityAssessments/ListByExtendedResourceServerVulnerabilityAssessments_example.json"
+                    }
+                },
+                "description": "Gets a list of server vulnerability assessment onboarding statuses on a given resource.",
+                "tags": [
+                    "ServerVulnerabilityAssessments"
+                ],
+                "operationId": "ServerVulnerabilityAssessment_ListByExtendedResource",
+                "parameters": [
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "name": "resourceNamespace",
+                        "in": "path",
+                        "description": "The Namespace of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceType",
+                        "in": "path",
+                        "description": "The type of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceName",
+                        "in": "path",
+                        "description": "Name of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ServerVulnerabilityAssessmentsList"
```

Comment:
>ServerVulnerabilityAssessmentsList [](start = 51, length = 34)

How many assessments would you expect in the worst case? Will you ever need paging for that?

Reply:
I saw the description. It looks like there's only one. Feel free to ignore this ccomment :)

---
In reply to: [281376642](https://github.com/Azure/azure-rest-api-specs/pull/5571#discussion_r281376642) [](ancestors = 281376642)